box.json

```diff
@@ -0,0 +1,29 @@
+{
+    "alias": "satis.phar",
+    "main": "bin/satis",
+    "output": "satis.phar",
+    "stub": true,
+    "directories": [ "src", "views" ],
+    "files": [
+        "LICENSE"
+    ],
+    "finder": [{
+        "name": "*.php",
```

Comment:
Actually, Satis isn't an interactive application, as a Phar or not.
The only use case for interactivity could be when asking for username/password when getting a _remote_ `satis.json` file. But it doesn't happen because the io is a NullIO Object, always (AFAIK).